NAMESPACE

```diff
@@ -1,14 +1,15 @@
 # Generated by roxygen2: do not edit by hand
 
 export(calendar_auth)
-export(calendar_auth_config)
+export(calendar_auth_configure)
 export(calendar_deauth)
 export(calendar_endpoints)
 export(calendar_events)
 export(calendar_has_token)
 export(calendar_list)
 export(calendar_oauth_app)
 export(calendar_token)
+export(calendar_user)
```

Comment:
I added a new function that is possible due to some newly exported helpers in gargle. Nice for figuring out who gcalendr thinks you are right now.